components/gitpod-db/src/tables.ts

```diff
@@ -268,6 +268,11 @@ export class GitpodTableDescriptionProvider implements TableDescriptionProvider
             deletionColumn: "deleted",
             timeColumn: "_lastModified",
         },
+        {
+            name: "d_b_user_ssh_public_key",
+            primaryKeys: ["id"],
+            timeColumn: "_lastModified",
```

Comment:
@mustard-mh I think we need `deletionColumn: "deleted",` here.
Sorry for the double-entry (cmp. `deleted-entry-gc.ts`). :cry: This is debt we need to get rid of...

Reply:
I use `env_var` as my baseline to change db related things, as we can see we lose a lot of `db-sync` feature (env_var have no `deletionColumn` too), not sure if it'll affect our data

https://github.com/gitpod-io/gitpod/blob/3c327432c8a22b93f3b3d46193bb1a2c84107a3a/components/gitpod-db/src/tables.ts#L171-L175

Reply:
We should properly change this file and baseline file [internal chat](https://gitpod.slack.com/archives/C02EN94AEPL/p1654843498128069?thread_ts=1654842003.970989&cid=C02EN94AEPL)

Reply:
@AlexTugarev Can you verify my comment above? :point_up:

Reply:
Synced with Alex, he confirmed. :+1: Will approve.